ProjectFinale/src/com/company/clase/Carte.java

```diff
@@ -0,0 +1,192 @@
+package com.company.clase;
+
+import com.company.clase.enumerari.TipCarte;
+import com.company.clase.enumerari.TipPersoana;
+import com.company.clase.mostenire.Autor;
+import com.company.clase.mostenire.Cititor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Carte {
```

Comment:
[Fix] ai doar 4 entitati, din minim 8 necesare (Carte, Persoana, Autor, Cititor), si acestea nu sunt POJOuri cum era cerinta fiindca vad ca ai amestecat si logica in aceste clase